app/assets/javascripts/discourse/tests/acceptance/plugin-outlet-parent-view-test.js

```diff
@@ -0,0 +1,29 @@
+import { acceptance, query } from "discourse/tests/helpers/qunit-helpers";
+import { hbs } from "ember-cli-htmlbars";
+import { test } from "qunit";
+import { visit } from "@ember/test-helpers";
+import { withSilencedDeprecationsAsync } from "discourse-common/lib/deprecated";
+import { registerTemporaryModule } from "discourse/tests/helpers/temporary-module-helper";
+
+acceptance("Plugin Outlet - Deprecated parentView", function (needs) {
+  needs.hooks.beforeEach(() => {
```

Comment:
```suggestion
  needs.hooks.beforeEach(function () {
```

We should set a good example 😛 (can't use `this` in arrow function hooks, so better to stick to this form)